src/config/dark_sites.json

```diff
@@ -1,13 +1,15 @@
 [
+    "alienwarearena.com",
     "animeonline.su",
     "asciinema.org",
+    "avengedsevenfold.com",
     "basho.com/$",
     "bbc.co.uk/iplayer",
     "beastskills.com",
     "bing.com/$",
     "bitcoinwisdom.com",
     "blizzard.com",
-    "blog.dota2.com",
+    "dota2.com",
```

Comment:
Begins now with "d", so move above `dotabuff.com` ([line 21](https://github.com/alexanderby/darkreader/pull/179/commits/eb7599f18d11c9dda88ad3752d0d7bb3b1d5744d#diff-e77420c8328c257db1f4f2db99e3de80R21)) to preserve alphabetic order.

Reply:
done, thanks